.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+name: Lint,Test,Deploy
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+
+jobs:
+  lint_test_deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Node setup
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'npm'
+      - name: install dependencies
+        run: npm ci
+
+      - name: lint
+        run: npm run lint:affected
+
+      - name: test
+        run:  npm run test:affected
+
+      - name: deploy
```

Comment:
I'm wondering about this step. We have the option to set the `stage` based on the `NODE_ENV` env parameter, for example, `NODE_ENV=stg npm run deploy`. The default value is `dev`.

Reply:
It means that we need to set `NODE_ENV=prod`

Reply:
Okay will do that

Reply:
hi @NetanelBasal have added  NODE_ENV=prod and also aws keys 
kindly check out the new changes